hack/tools/release/notes/test/golden/v1.5.0.md

```diff
@@ -57,9 +57,9 @@ REPLACE ME: A couple sentences describing the deprecation, including links to do
 - CAPD: Add kind mapper (#8880)
 - CAPD: Change the haproxy entrypoint to prevent getting stopped immediately after start (#8685)
 - CAPD: Delegate CAPD port selection to the container runtime (#8642)
+- CAPD: Fix fail-swap-on=false flag not being part of kind images anymore (#8767)
 - CAPD: Implement watch filter (#8789)
 - CAPD: Test/capd: fix kind mapper entry for v1.25.11 (#8914)
-- CAPD: Test/e2e fix fail-swap-on=false flag not being part of kind images anymore (#8767)
```

Comment:
These the same PR. The title was changed in the PR and since now we read from the PR API and not the commits, now we output the new title. This is expected (and IMO a good change).